src/grid-extender/columns-provider.ts

```diff
@@ -23,6 +23,7 @@ class DynamicItemIndexColumnsProvider implements ListColumnsProvider {
         const column: ColumnModel = {
             name: this.columnName,
             title: this.columnTitle,
+            ordinal: 50,
```

Comment:
Without this, the custom columns are placed last. Please, write a test, that verifies the column is adjusted based on the ordinal.